tests/scripts/build-release.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -ex
+
+#############
+# FUNCTIONS #
+#############
+
+function  build() {
+    # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+    build/run make VERSION=0 build.all
+    # quick check that go modules are tidied
+    build/run make mod.check
+}
+
+if echo "${BRANCH_NAME}" | grep "release-" ; then
+    build
+    build/run make -C build/release build BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true GIT_API_TOKEN=${GIT_PSW}
+    git status &
```

Comment:
Why do you run the command in the background?

Reply:
When I was testing... screen holding/stuck in this(don't remember the exact problem) but there was something.

Reply:
Did you try removing this? This shouldn't be necessary.

Reply:
No, because anyway we need this later for the release branch...so I kept it.

Reply:
What do you "we were doing it?" I don't see it in the Jenkins file

Reply:
So this is for checking master and release. We can remove this for now as this PR is all about master but later we need this when doing for release.

Reply:
Let's revisit this after this merge.